### ISSUE TITLE
Permissions about created files with cloud9-docker

### ISSUE THREAD

**Issue by huskier** [kdelfour/cloud9-docker#3]
Dear Kevin,
I deployed your latest cloud9-docker(V3) on my server, and on the server side I run the following command:
sudo docker run -it -d -p 8181:8181 -v /mnt/development/:/workspace/ kdelfour/cloud9-docker

then, I access the editor from other computers. The problem is that the owner of the files which I created is "root", and the permission session reads like:
-rw-r--r-- 1 root root 0 Mar  1 09:47 testfile.txt

This is not what I want. I just want to create the files with a general user.

Could you give me a hand on this issue?

Thanks!

Simon

**Comment by jslucas** [comment]
dear kevin,
i have the same issue, can u help us?

**Comment by kdelfour** [comment]
I'm going to try to find a solution but if you want to create files with a general user, you have to create the same user in the container. 

You can override this dockerfile and add this : 

       RUN adduser --system --group --shell /bin/sh auser && mkdir /home/auser/bin
       USER auser

Before that, you create and chown the volume with the same user before bind-mounting it, it will work.

Basically, the user in container is "root".
